expected/array_spread.out

```diff
@@ -0,0 +1,13 @@
+do $$
+  Object.prototype[Symbol.iterator] = function() {
+     return {
+       next:() => this
+     }
+  };
+  [...({})];
+$$ language plv8;
+FATAL:  plv8: OOM error in GC
+server closed the connection unexpectedly
```

Comment:
I wonder if this might be better served by throwing an error, since it is specifically a javascript OOM error, not a full server OOM.

```
throw js_error("OOM error");
```

Reply:
this is what happens with `js_error` or any other non-fatal thing:
```
terminate called after throwing an instance of 'js_error'
2019-05-24 11:08:43.852 EEST [21345] LOG:  server process (PID 21389) was terminated by signal 6: Aborted
2019-05-24 11:08:43.852 EEST [21345] DETAIL:  Failed process was running: do $$ 
          Object.prototype[Symbol.iterator] = function() {
             return {
               next:() => this
             }
          };
          [...({})]; 
        $$ language plv8
        ;
2019-05-24 11:08:43.853 EEST [21345] LOG:  terminating any other active server processes
```

Reply:
okay, found out another hook, that tremendously improved the robustness 
now it can safely throw `js_error`